spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -97,7 +97,7 @@ public TaskExecutionController(TaskExplorer explorer, TaskService taskService,
 	@ResponseStatus(HttpStatus.OK)
 	public PagedResources<TaskExecutionResource> list(Pageable pageable,
 			PagedResourcesAssembler<TaskJobExecutionRel> assembler) {
-		Page<TaskExecution> taskExecutions = explorer.findAll(pageable);
+		Page<TaskExecution> taskExecutions = this.taskService.findAll(pageable);
```

Comment:
1) Good question.  Usually I leave business logic outside of a controller.  In this case the masking was put in the TaskService.  I can move the logic into the controller, but what is wrong with the methods being in the TaskService?
2) To the second question, Spring Cloud Task shouldn't have a TaskExplorer that is specific to a particular app, in this case a password redaction.  This should be handled by the app that is using itself.